### PR TITLE
Feat/k8s-1.31.4

### PR DESCRIPTION
**Pull request by ralgozino**
- Change 1.31 version from 1.31.3 to 1.31.4 that has been release while we were working on this update
- Update all docs preparing for the release
- Fix some outdated info in docs and examples

### CODE REVIEW

File: examples/playbooks/furyagent.yml
Comment (nutellinoit) [inline examples/playbooks/furyagent.yml:1]: I think that this file is needed by furyctl nevertheless? Even if we use furyctl create pki?

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Overall LGTM with two comments below 👇

File: examples/playbooks/furyagent.yml
Comment (ralgozino) [reply examples/playbooks/furyagent.yml:1]: nope

**Review by nutellinoit — APPROVED** [review]
LGTM!

**Pull request merged**